draft-birkholz-rats-corim.md

```diff
@@ -789,10 +793,629 @@ Not included in the full CDDL definition are CDDL dependencies to CoSWID. The fo
+
+Future registrations for this registry are to be made based on {{RFC8126}} as follows:
+
+| Range             | Registration Procedures
+|---
+| 0-127    | Standards Action
+| 128-255  | Specification Required
+{: #tbl-iana-corim-map-items-reg-procedures title="CoRIM Map Items Registration Procedures"}
+
+All negative values are reserved for Private Use.
+
+Initial registrations for the "CoRIM Map" registry are provided below.
+Assignments consist of an integer index value, the item name, and a reference to the defining specification.
+
+| Index | Item Name | Specification
+|---
+| 0 | corim.id | RFC-AAAA
+| 1 | corim.tags | RFC-AAAA
+| 2 | corim.dependent-rims | RFC-AAAA
+| 3-255 | Unassigned
+{: #tbl-iana-corim-map-items title="CoRIM Map Items Initial Registrations"}
+
+{: #iana-corim-locator-map-items}
+## CoRIM Locator-Map Items Registry
+
+This document defines a new registry titled "CoRIM Locator Map".
+The registry uses integer values as index values for items in 'corim-locator-map' CBOR maps.
+
+Future registrations for this registry are to be made based on {{RFC8126}} as follows:
+
+| Range             | Registration Procedures
+|---
+| 0-127    | Standards Action
+| 128-255  | Specification Required
+{: #tbl-iana-corim-locator-map-items-reg-procedures title="CoRIM Locator Map Items Registration Procedures"}
+
+All negative values are reserved for Private Use.
+
+Initial registrations for the "CoRIM Locator Map" registry are provided below.
+Assignments consist of an integer index value, the item name, and a reference to the defining specification.
+
+| Index | Item Name | Specification
+|---
+| 0 | corim.href | RFC-AAAA
+| 1 | corim.thumbprint | RFC-AAAA
+| 2-255 | Unassigned
+{: #tbl-iana-corim-locator-map-items title="CoRIM Locator Map Items Initial Registrations"}
+
+{: #iana-corim-validity-map-items}
+## CoRIM Validity-Map Items Registry
+
+This document defines a new registry titled "CoRIM Validity Map".
+The registry uses integer values as index values for items in 'validity-map' CBOR maps.
+
+Future registrations for this registry are to be made based on {{RFC8126}} as follows:
+
+| Range             | Registration Procedures
+|---
+| 0-127    | Standards Action
+| 128-255  | Specification Required
+{: #tbl-iana-corim-validity-map-items-reg-procedures title="CoRIM Validity Map Items Registration Procedures"}
+
+All negative values are reserved for Private Use.
+
+Initial registrations for the "CoRIM Validity Map" registry are provided below.
+Assignments consist of an integer index value, the item name, and a reference to the defining specification.
+
+| Index | Item Name | Specification
+|---
+| 0 | corim.not-before | RFC-AAAA
+| 1 | corim.not-after | RFC-AAAA
+| 2-255 | Unassigned
+{: #tbl-iana-corim-validity-map-items title="CoRIM Validity Map Items Initial Registrations"}
+
+{: #iana-corim-meta-map-items}
+## CoRIM Meta-Map Items Registry
+
+This document defines a new registry titled "CoRIM Meta Map".
```

Comment:
corim-meta-map isn't extensible today. As a 'place' for metadata about the manifest it reasonably should be extensible (under the control of a standards body). Possibly, define it as not allowing private use? If these questions require more discussion we should opt to leave it not extensible for now.

Reply:
If we are talking about elements of the IM that should be extensible I think this deserves at least a mention :-)
Also, I wouldn't discourage private use.